aggsender/db/aggsender_db_storage.go

```diff
@@ -564,18 +578,33 @@ func (a *AggSenderSQLStorage) GetNonAcceptedCertificate() (*NonAcceptedCertifica
 	if err := json.Unmarshal([]byte(val), &nonAcceptedCert); err != nil {
 		return nil, fmt.Errorf("failed to unmarshal non-accepted certificate: %w", err)
 	}
+	if strings.HasPrefix(nonAcceptedCert.SignedCertificate, prefixFilename) {
+		// The content is pointing to a file
+		certificateFilePath := nonAcceptedCert.SignedCertificate[1:]
+		data, err := os.ReadFile(certificateFilePath)
+		if err != nil {
+			return nil, fmt.Errorf("getNonAcceptedCertificate: failed to read signed certificate file %s: %w",
+				certificateFilePath, err)
+		}
+		certHash := crypto.Keccak256Hash([]byte(string(data)))
```

Comment:
It should be as simple as this :slightly_smiling_face:

```suggestion
		certHash := crypto.Keccak256Hash(data)
```

Reply:
good point!, thanks you!